### ISSUE TITLE
Npcap - no raw 802.11 packets captured on Win10

### ISSUE THREAD

**Issue by pr0mixe** [nmap/nmap#1877]
Hello,
I've had this problem for a couple of months now, been waiting for newer versions of Npcap, but the lastest version 0.9986 still has the same issue:
Npcap works fine capturing _regular data packets_, but as soon as I switch to the Monitor Mode, either inside Wireshark or via wlanhelper.exe, there are no packets received by Wireshark, the capture window stays blank. Tried it on 3 (!) different Windows 10 machines (two x64, one x86), including a brand new clean Win10 install. Tried with two USB 802.11 interfaces that have been working fine on other machines. Tried all versions of Npcap between 0.991 and 0.9986 - nothing brings back the Monitor Mode packet capture... Tried Wireshark versions from 3.0.3 to 3.2.0. 
I thought that maybe starting with Win10 1903 something got broken because I haven't been able to see 802.11 packets on multiple installs of 1903, but the latest puzzle is one of the machines being 1809 and the same issue exists. But my other 1809 machine works just fine %%, been using Npcap 0.9981 & Wireshark 3.0.6 and with the same USB adapters as above I can see raw 802.11 packets as expected.
How do I diagnose this issue? Any logs to look into? 
To recap:
1) Npcap IS working when capturing regular data, not raw 802.11
2) Monitor Mode DOES get enabled, but no packets are received.
3) Same versions of Npcap/WS with same hardware and OS DO work on some other machines, but not universally on every machine..
Thanks!

**Comment by pr0mixe** [comment]
Following up on this issue:
I'm attaching two DiagReports from _identical_ computers. The "lous" works fine, captures raw 802.11 packets. The "euclid" _doesn't_ capture raw 802.11 packets, using same WiFi adapter, same driver, same versions of Windows, Wireshark, Npcap, etc. This is driving me nuts! 
Where should I look for answers? What is the next step to troubleshoot? Where do I look? Please help!

[DiagReport-20200116-223333-lous.txt](https://github.com/nmap/nmap/files/4079777/DiagReport-20200116-223333-lous.txt)
[DiagReport-20200116-225919-euclid-995.txt](https://github.com/nmap/nmap/files/4079778/DiagReport-20200116-225919-euclid-995.txt)
